src/main/java/teammates/common/util/Const.java

```diff
@@ -924,6 +916,7 @@ private Const() {
         public static final String ERROR_FEEDBACK_EMAIL_SUBJECT = "errorfeedbackemailsubject";
         public static final String ERROR_FEEDBACK_EMAIL_CONTENT = "errorfeedbackemailcontent";
         public static final String ERROR_FEEDBACK_URL_REQUESTED = "errorfeedbackrequestedurl";
+        public static final String ENTITY_TYPE = "entitytype";
```

Comment:
Should we define `ENTITY_TYPE_STUDENT = "student"` or `ENTITY_TYPE_INSTRUCTOR = "instructor"` here?